tools/topology/sof/pipe-codec-adapter-playback.m4

```diff
@@ -0,0 +1,112 @@
+# Low Latency Passthrough Pipeline and PCM
+#
+# Pipeline Endpoints for connection are :-
+#
+#  host PCM_P --> B0 --> CODEC_ADAPTER -> B1 --> sink DAI0
+
+# Include topology builder
+include(`utils.m4')
+include(`buffer.m4')
+include(`pcm.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`codec_adapter.m4')
+include(`bytecontrol.m4')
+
+#
+# Controls
+#
+
+# For codec developers, please define the following control bytes of setup config and runtime params
+# and the corresponding max bytenum on your own.
```

Comment:
Can we have more context in the comment here e.g. what tools would be used, any example. just basic info to help put folks on teh right track. btw, should these headers be all 0s as "empty" controls ?

Reply:
I added some contexts on it, do you think there is any information needed to append?

> btw, should these headers be all 0s as "empty" controls ?

I would prefer a valid but dummy default here, "empty" also sounds good to me.

Reply:
ok - as long as this is in the comments so it's easy to follow.